diff --git a/src/sbas.c b/src/sbas.c
index 5278b6dc7ca7f4aab2d68108ed5d8f875647fa18..bcc63ec6788c90a053772c4aa50ac317c5419ad9 100644
--- a/src/sbas.c
+++ b/src/sbas.c
@@ -65,7 +65,7 @@ x6[]={-180,-170,-160,-150,-140,-130,-120,-110,-100,- 90,- 80,- 70,- 60,- 50,
 x7[]={-180,-150,-120,- 90,- 60,- 30,   0,  30,  60,  90, 120, 150},
 x8[]={-170,-140,-110,- 80,- 50,- 20,  10,  40,  70, 100, 130, 160};
 
-EXPORT const sbsigpband_t igpband1[9][8]={ /* band 0-8 */
+const sbsigpband_t igpband1[9][8]={ /* band 0-8 */
     {{-180,x1,  1, 28},{-175,x2, 29, 51},{-170,x3, 52, 78},{-165,x2, 79,101},
      {-160,x3,102,128},{-155,x2,129,151},{-150,x3,152,178},{-145,x2,179,201}},
     {{-140,x4,  1, 28},{-135,x2, 29, 51},{-130,x3, 52, 78},{-125,x2, 79,101},
@@ -85,7 +85,7 @@ EXPORT const sbsigpband_t igpband1[9][8]={ /* band 0-8 */
     {{ 140,x3,  1, 27},{ 145,x2, 28, 50},{ 150,x3, 51, 77},{ 155,x2, 78,100},
      { 160,x3,101,127},{ 165,x2,128,150},{ 170,x3,151,177},{ 175,x2,178,200}}
 };
-EXPORT const sbsigpband_t igpband2[2][5]={ /* band 9-10 */
+const sbsigpband_t igpband2[2][5]={ /* band 9-10 */
     {{  60,x5,  1, 72},{  65,x6, 73,108},{  70,x6,109,144},{  75,x6,145,180},
      {  85,x7,181,192}},
     {{- 60,x5,  1, 72},{- 65,x6, 73,108},{- 70,x6,109,144},{- 75,x6,145,180},
diff --git a/src/options.c b/src/options.c
index a64b12e315a276be63b34abb98adb331ed825778..a33a2c8d5a077c55eec767eac1e8009a2d56d259 100644
--- a/src/options.c
+++ b/src/options.c
@@ -63,7 +63,7 @@ static char snrmask_[NFREQ][1024];
 #define TIDEOPT "0:off,1:on,2:otl"
 #define PHWOPT  "0:off,1:on,2:precise"
 
-EXPORT opt_t sysopts[]={
+opt_t sysopts[]={
     {"pos1-posmode",    3,  (void *)&prcopt_.mode,       MODOPT },
     {"pos1-frequency",  3,  (void *)&prcopt_.nf,         FRQOPT },
     {"pos1-soltype",    3,  (void *)&prcopt_.soltype,    TYPOPT },
